app/allocation/components/FarcasterModal.tsx

```diff
@@ -12,6 +12,9 @@ interface FarcasterModalProps {
 }
 
 const FarcasterModal: React.FC<FarcasterModalProps> = ({ isOpen, onClose }) => {
+  if(!isOpen) {
+    return null;
```

Comment:
I am not sure why the splash page is auto-scrolling upwards to the top but It is happening when FarcasterModal and WorldIdSignInModal even though it isnt open is calling the /delegate/status and /user/badges and when it is returning an error it os going to the top invoked by NoSSRModals.

hence I added this